project/Dependencies.scala

```diff
@@ -21,7 +21,7 @@ object Dependencies {
 
   val bijectionCoreV = "0.9.7"
   val collectionCompatV = "2.8.0"
-  val coursierV = "2.0.0-RC5-6"
```

Comment:
this is actually the latest version available also for scala 2.11 (see https://github.com/scalacenter/scalafix/runs/8031777807?check_suite_focus=true) Hence I think we can use 2 different versions of coursier instead, one for 2.11 and the other for the rest

Reply:
let's not bump coursier and introduce more complexity, 2.11 is on its way out anyway https://github.com/scalacenter/scalafix/issues/1635

Reply:
reverted